src/ndarray/ndarray.cc

```diff
@@ -614,8 +614,7 @@ NDArray &NDArray::operator/=(const real_t &src) {
 }
 
 void NDArray::Save(dmlc::Stream *strm) const {
-  // save shape
-  shape_.Save(strm);
```

Comment:
@tqchen Does ndarray dupm files have a version system? Can we store new arrays as int64 while supporting legacy read from int32?